src/devtools/client/debugger/src/client/commands.js

```diff
@@ -339,12 +339,13 @@ async function getSourceActorBreakpointHitCounts({ id }, lineNumber, onFailure)
   // explanation of the bounds here.
   const lowerBound = Math.floor(lineNumber / MAX_LINE_HITS_TO_FETCH) * MAX_LINE_HITS_TO_FETCH;
   const upperBound = lowerBound + MAX_LINE_HITS_TO_FETCH;
-  const locationsToFetch = locations.filter(
-    location => location.line >= lowerBound && location.line < upperBound
-  );
+  const locationsToFetch = locations
+    .filter(location => location.line >= lowerBound && location.line < upperBound)
+    .map(location => ({ ...location, columns: [location.columns.sort((a, b) => a - b)[0]] }));
```

Comment:
We'll always have at least one item in the `columns` array? It's an `any` type so I'm not sure.

Reply:
Yeah, the backend won't return the line as an entry if there is not at least one breakable column.